tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -0,0 +1,724 @@
+                    .build();
+
+            FileSystemSecurityRealm newFileSystemRealm = FileSystemSecurityRealm.builder()
+                    .setRoot(Paths.get(descriptor.getOutputRealmLocation(), descriptor.getFileSystemRealmName()))
+                    .setSecretKey(key)
+                    .setLevels(descriptor.getLevels())
+                    .setProviders(ELYTRON_PASSWORD_PROVIDERS)
+                    .build();
+            FileSystemRealmUtil.createEncryptedRealmFromUnencrypted(oldFileSystemRealm, newFileSystemRealm);
+        }
+    }
+
+    /**
+     * Creates the script/commands the user must run for Elytron to recognize
+     * and use the new filesystem-realm
+     */
+
+    private void createWildFlyScript() throws Exception {
+        int counter = 0;
+        for (Descriptor descriptor : descriptors) {
+            String inputRealmLocation = descriptor.getInputRealmLocation();
+            String outputRealmLocation = descriptor.getOutputRealmLocation();
+            String fileSystemRealmName = descriptor.getFileSystemRealmName();
+            String credentialStore = descriptor.getCredentialStore();
+            String secretKeyAlias = descriptor.getSecretKeyAlias();
+            int levels = descriptor.getLevels();
+
+            if(secretKeyAlias == null) {
+                secretKeyAlias = "key";
+            }
+            String createScriptCheck = "";
+            if (Paths.get(String.format("%s.sh", fileSystemRealmName)).toFile().exists()) {
+                createScriptCheck = prompt(
+                        false,
+                        null,
+                        false,
+                        ElytronToolMessages.msg.shouldFileBeOverwritten(String.format("%s.sh", fileSystemRealmName))
+                );
+            }
+            String fullOutputPath;
+            if (outputRealmLocation.startsWith(".")) {
+                fullOutputPath = Paths.get(outputRealmLocation.substring(2)).toAbsolutePath().toString();
+            } else {
+                fullOutputPath = Paths.get(outputRealmLocation).toAbsolutePath().toString();
+            }
+
+            if (summaryMode) {
+                summaryString.append(String.format("Configured script for WildFly named %s.sh at %s.", fileSystemRealmName, fullOutputPath));
+                summaryString.append(System.getProperty("line.separator"));
+                summaryString.append("The script is using the following names:");
+                summaryString.append(System.getProperty("line.separator"));
+                summaryString.append(String.format("Name of filesystem-realm: %s", fileSystemRealmName));
+                summaryString.append(System.getProperty("line.separator"));
+            }
+
+            List<String> scriptLines = Arrays.asList(
+                String.format("/subsystem=elytron/secret-key-credential-store=%s:add(path=%s, create=true)", "mycredstore"+counter, credentialStore),
```

Comment:
The Elytron tool command should have either generated the credential store file for the user or made use of an existing user-specified credential store file. The `create` attribute can just be removed here. The file will exist anyway.

Reply:
Instead of mycredstore, we should be using the user-specified value.